src/cart/utils/crt_utils.c

```diff
@@ -454,10 +457,6 @@ crtu_dc_mgmt_net_cfg_setenv(const char *name, crt_init_options_t *opt)
 
 cleanup:
 	dc_put_attach_info(&crt_net_cfg_info, crt_net_cfg_resp);
-	D_FREE(opt->cio_provider);
-	D_FREE(opt->cio_interface);
-	D_FREE(opt->cio_domain);
```

Comment:
this caused opt values to be null by the time crt_init() was called in crtu_cli_start_basic()